benchmarks/blind_v8/external_001/oracles/V8-004/oracle.py

```
@@ -0,0 +1,132 @@
+import tempfile
+import pytest
+from doubled_lines import main
+
+@pytest.fixture
+def sample_file(request):
+    """
+    Fixture to create a temp file with given content. Returns the filename and ensures cleanup.
+    """
+    files = []
+    def _makefile(content, encoding='utf-8', mode='w'):
+        fd, fname = tempfile.mkstemp()
```

Ruff (0.16.2)

[warning] Unpacked variable `fd` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)